src/neural/blas/network_blas.cc

```diff
@@ -134,13 +147,21 @@ class BlasComputation : public NetworkComputation {
   ActivationFunction ffn_activation_;
   bool attn_policy_;
   bool attn_body_;
+
+  static std::mutex buffers_lock_;
```

Comment:
Why making them static rather than the member variable of `BlasNetwork`?
With static, it will not work if e.g. we create two instances of BlasNetwork (happens during selfplay, for example).